src/main/java/org/broadinstitute/consent/http/db/DAOContainer.java

```diff
@@ -173,4 +174,8 @@ public WorkspaceAuditDAO getWorkspaceAuditDAO() {
   public void setWorkspaceAuditDAO(WorkspaceAuditDAO workspaceAuditDAO) {
     this.workspaceAuditDAO = workspaceAuditDAO;
   }
+
+  public InstitutionDAO getInstitutionDAO() { return institutionDAO; }
+
+  public void setInstitutionDAO(InstitutionDAO institutionDAO) { this.institutionDAO = institutionDAO; }
```

Comment:
This is fine. I created this class because Codacy was complaining about too many arguments in a constructor. I'm pretty sure you don't need this class in the usages of `DAOContainer`, but it is nice to have it if needed.